packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -12,7 +12,7 @@ import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {View} from "@khanacademy/wonder-blocks-core";
 
 import type {StyleType} from "@khanacademy/wonder-blocks-core";
-import visibilityModifierDefaultConfig from "../util/visibility-modifier.js";
+import visibilityModifierDefaultConfig from "../../wonder-blocks-core/util/visibility-modifier.js";
```

Comment:
This results in the code appearing in multiple packages and results in packages being larger than they need to be.  Please export this from the `wonder-blocks-core` package.

Reply:
>  Since it's only used internally, I didn't add this to the exports of core, but rather each package using it just grabs it from there. This seems reasonable to me and MUCH better than duplicating this code.

I should read the summary before requesting changes.  I wonder if it makes sense to have a `wonder-blocks-internal` that's used for this kind of stuff so that we can avoid these anti-patterns.  The reason why I'm against requiring files directly is that it can lead to packages being larger (and in some case much larger) than they need to be.  In a previous PR I cleaned up a number of places where we were doing this.

Reply:
Yes, I understand that. In this case, it felt like the right move, but we could add an internals package or just accept this as an export of core. The issue I have with either of those is that once something is on NPM, it's public and therefore its API is public; I don't know that we want these things to be public at all.

Reply:
I see the concern.  Paying for `npm` to have a single private `npm` module seems like overkill.  Maybe we could just have a `internal` folder that lives outside of `packages` for this reason.  I'm not really sure the best way to address this so let's go with this for now.  Please add a comment to the import to discourage people from using this pattern except when it's absolutely necessary.  I want to avoid bloating our packages at worse and or ending up with circular dependencies at worst.

Reply:
I think we can do two things here and I'll do both.

1) Move the files in question to an internal-only folder
2) Comment on the places they are imported

That way we can eventually lint for this and ensure that only files in an `internal-only` folder are used this way, which will help us identify it during reviews and discourage it from happening.

Reply:
@kevinbarabash Looks like this eslint rule would work. Any issues with adding this library of rules as a dev dependency?

https://github.com/benmosher/eslint-plugin-import/blob/HEAD/docs/rules/no-restricted-paths.md

Reply:
Go for it.  I wasn't sure how to use it to enforce the constraint we want, but maybe there's a way to have different restricted paths for each package.